### DISCUSSION TITLE
How stable is "preview"?

### DISCUSSION THREAD

**Discussion by mjgp2** [getredash/redash#6735] [Q&A]
Hi all,

I'm looking at deploying out redash, and it seems to me like the "preview" build on master seems akin to a release candidate? Wondering how stable this is, and if we are planning to deploy in the next couple of months the probability of having a release off the master branch?

Thanks so much 😄

**Comment by guidopetri** [comment]
It's sort of a release candidate, yeah. It hasn't been fully tested and we're starting to think about how to build a proper release, but we don't have any specific plans nor a timeline yet.

**Comment by eradman** [comment]
Closing in favor of https://github.com/getredash/redash/discussions/6647
